src/PrestaShopBundle/Resources/config/routing/admin/sell/orders/orders.yml

```
@@ -191,6 +191,26 @@ admin_orders_update_invoice_note:
     orderId: \d+
     orderInvoiceId: \d+
 
+admin_orders_split_shipment:
+  path: /{orderId}/shipment/{shipmentId}/split
+  methods: [ POST ]
+  defaults:
+    _controller: PrestaShopBundle\Controller\Admin\Sell\Order\OrderController::splitShipmentAction
+  requirements:
+    orderId: \d+
+    shipmentId: \d+
+
+admin_orders_shipment_get_split_form:
+  path: /shipment/get-shipment-split-form
```

can we have something close to the similar route for the merge?

```suggestion
  path: /{orderId}/shipment/split-form
```